### PR TITLE
Blog Post 19th March 2021

### PR DESCRIPTION
**Pull request by pmcmaw**

### CODE REVIEW

File: _posts/2021-03-19-status-update.md [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+categories:
+  - team
+  - status
+  - community
+  - holidays
+  - releases
+  - modules
+  - gems
+  - ci
+tags:
```
Comment (DavidS) [inline _posts/2021-03-19-status-update.md:14]:
```suggestion
tags:
  - community
  - holidays
  - releases
  - modules
  - gems
  - ci
```

File: _posts/2021-03-19-status-update.md [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+
+During the work for the multi-node testing supported we have uncovered a small issue that was related to the created firewall rule persistence in debugging sessions and we'll have a fix for this as soon as possible. Work is still [in progress](https://github.com/puppetlabs/puppetlabs-websphere_application_server/pull/223) on the [puppetlabs-websphere_application_server][puppetlabs-websphere_application_server] module and we are hopeful to find a solution soon.
+
+## Terminology Changes
+
+All of our terminology work is coming to a close. What this means is once it is completely finished we will start **deleteing 'master' branches in our repos**. If you are pinning to this, I advise you to address this as soon as you possibly can. This work effort has been carried out on both our modules and tools. Shout out to [David Swan][DavidSwan] for heading up this work.
```
Comment (DavidS) [inline _posts/2021-03-19-status-update.md:77]:
```suggestion
All of our terminology work is coming to a close. What this means is once it is completely finished we will start **deleting 'master' branches in our repos**. If you are pinning to this, I advise you to switch over to `main` as soon as you possibly can. This work effort has been carried out on both our modules and tools. Shout out to [David Swan][DavidSwan] for heading up this work.
```

File: _posts/2021-03-19-status-update.md (hunk 1 above)
Comment (pmcmaw) [reply _posts/2021-03-19-status-update.md:14]: Thanks, I literally have no words for what I done there. :-|

Comment (DavidS) [reply _posts/2021-03-19-status-update.md]: it happens :shrug:

**Review by david22swan — APPROVED** [review]

**Review by sheenaajay — APPROVED** [review]

**Pull request merged**